### ISSUE TITLE
Version Solving failed

### ISSUE THREAD

**Issue by divyanshub024** [simc/crimson#1]
### Log 

```
crimson_demo % flutter pub get
Because every version of flutter from sdk depends on collection 1.16.0 and analyzer >=5.3.1 depends on collection ^1.17.0, flutter from sdk is incompatible with analyzer >=5.3.1.
And because every version of crimson depends on analyzer ^5.3.1, flutter from sdk is incompatible with crimson.
So, because crimson_demo depends on both flutter from sdk and crimson ^0.0.2, version solving failed.
Running "flutter pub get" in crimson_demo...                            
pub get failed (1; So, because crimson_demo depends on both flutter from sdk and crimson ^0.0.2, version solving failed.)
```

**Comment by Abdulrasheed1729** [comment]
I'm also having the same issue here too.

**Comment by simc** [comment]
Should be fixed in 0.0.3

**Comment by Abdulrasheed1729** [comment]
But I later ran `flutter update-packages --force-upgrade` to upgrade the unresolved dependencies and it works
